ui/perfherder/alerts/AlertModal.jsx

```diff
@@ -90,14 +92,31 @@ export default class AlertModal extends React.Component {
             </FormGroup>
           </ModalBody>
           <ModalFooter>
-            <Button
-              color="secondary"
-              onClick={(event) => updateAndClose(event, inputValue)}
-              disabled={invalidInput || !inputValue.length || !validated}
-              type="submit"
-            >
-              Assign
-            </Button>
+            {regression && (
+              <Button
+                color="secondary"
+                onClick={(event) =>
+                  updateAndClose(event, inputValue, regression)
```

Comment:
We're calling `updateAndClose()` with 3 params, but it's actually defined to accept only 2.
So, what's happening here?